src/routes/(authentication)/login/+page.svelte

```diff
@@ -7,32 +7,39 @@
   import { branding, features } from '$lib/configuration';
 
   const redirectTo = $page.url.searchParams.get('redirectTo') || '/';
+  const siteName = branding.applicationName;
+  const greeting = branding.login.greeting;
+  const description = branding.login.description;
+  const openPicsureLinkText = branding.login.openPicsureLinkText;
+
   let selected: string;
-  const siteName = branding.applicationName || 'PIC-SURE Demo';
-  const title = branding.login.title || 'Welcome to PIC-SURE';
-  const description = branding.login?.description;
-  const openPicsureLinkText = branding.login.openPicsureLinkText || 'Explore without Login';
```

Comment:
What is the purpose of removing the back up defaults?

Reply:
I didn't want to hand-hold the users on the configuration page, which should always contain their customizations or the default values.